src/draw.jl

```diff
@@ -103,16 +99,8 @@ function layoutplot!(scene, layout, ts::ElementOrList)
         # dodge may need to be done separately per each subplot
         Ndodge = max(Ndodge, rank(to_value(get(spec.options, :dodge, Ndodge))))
     end
-    axs = facetlayout[1:Ny, 1:Nx] = [Axis(scene) for i in 1:Ny, j in 1:Nx]
-    for i in 1:Nx
-        linkxaxes!(axs[:, i]...)
-    end
-    for i in 1:Ny
-        linkyaxes!(axs[i, :]...)
-    end
-    hidexdecorations!.(axs[1:end-1, :], grid = false)
-    hideydecorations!.(axs[:, 2:end], grid = false)
-
+    # What is a better placeholder here, in case some plots are missing?
+    axs = Union{Axis, Nothing}[nothing for i in 1:Ny, j in 1:Nx]
```

Comment:
ggplot displays axes even if there is no data to display. This works since all axes are linked by default.

![](http://zevross.com/blog/wp-content/uploads/2019/03/facet_plot_5_grid-768x549.png)

Only for facet_wrap (which we don't have), it doesn't show empty axes. 


![](http://zevross.com/blog/wp-content/uploads/2019/03/facet_plot4_wrap2varstriptext.png)

Reply:
Yes, I've updated it to always start with an axis placeholder, I agree with you that it makes more sense.